code/game/dna/genes/monkey.dm

```diff
@@ -37,8 +37,8 @@
 
 
 	var/mob/living/carbon/human/monkey/O = null
-	if(H.species.primitive)
-		O = new H.species.primitive(src)
+	if(H.species.primitive_form)

```

Comment:
*That's even weirder.*  If there's *no* variable like that, it should throw an error on compile unless someone specifically used a colon to tell it to trust the var will exist.